russstation/code/controllers/subsystem/ticker.dm

```diff
@@ -0,0 +1,13 @@
+var/initialtpass = 0
+
+/datum/controller/subsystem/ticker/proc/votetimer()
+	var/timerbuffer = 0
+	if(initialtpass == 0)
+		timerbuffer = CONFIG_GET(number/transfer_delay_initial)
+	else
+		timerbuffer = CONFIG_GET(number/transfer_delay_interval)
```

Comment:
this is causing a runtime for some reason I haven't figured out yet

Reply:
Discuss past rounds

Reply:
`runtime error: Maximum recursion level reached (perhaps there is an infinite loop)
To avoid this safety check, set world.loop_checks=0.
proc name: Get (/datum/controller/configuration/proc/Get)
  usr: null
  src: Configuration (/datum/controller/configuration)
  call stack:
Configuration (/datum/controller/configuration): Get(/datum/config_entry/number/tra... (/datum/config_entry/number/transfer_delay_interval))
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
...
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): votetimer()
Ticker (/datum/controller/subsystem/ticker): setup()
Ticker (/datum/controller/subsystem/ticker): fire()
Ticker (/datum/controller/subsystem/ticker): ignite(0)`

Reply:
okay just for future documentation this was caused by trying to use recursion and spawn() operators. That's a bad idea in dm it turns out and its much better to do timers and what not in the fire() section of a subsystem using world.time.